src/color-picker/color-picker.scss

```diff
@@ -6,10 +6,10 @@
 
 $iui-color-picker-content-width: ($iui-l * 9) + ($iui-s * 8); // (9 swatches per row) + (gaps)
 $iui-color-picker-max-width: ($iui-l * 10) + ($iui-s * 9); // (10 swatches per row) + (gaps)
-$iui-color-swatch-border-radius: 5px;
-$iui-inset-box-shadow: inset 0 0 0 1px rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5));
-$iui-active-box-shadow: 0 0 0 $iui-xxs rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-3));
-$iui-hover-box-shadow: 0 0 1px $iui-xxs + 1 rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5));
+$iui-color-swatch-border-radius: $iui-border-radius * 2;
+$iui-inset-box-shadow: inset 0 0 0 1px rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-5));
+$iui-active-box-shadow: 0 0 0 $iui-xxs rgba(var(--iui-color-foreground-primary-rgb), var(--iui-opacity-3));
+$iui-hover-box-shadow: 0 0 1px $iui-xxs + 1 rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-5));
```

Comment:
If you're using `$iui-xxs + 1` instead of `3px`, why not use `$iui-xxs - 1` instead of 1px?

Reply:
These haven't changed compared to before

Reply:
Right now our spacing variables are all hard coded.  I;d like for these variables to be replaced with REM which will all scale relative to `$iui-m`.  So for example, if we switched `$iui-m` from 16px to 14px, then `$iui-l` ($iui-m * 1.5) would change from 24px to 21px.

Borders on the other hand, I think we'd want them to stay 1px no matter what.